ext/serializer.c

```diff
@@ -1504,6 +1505,10 @@ void ddtrace_serialize_span_to_array(ddtrace_span_data *span, zval *array) {
     zend_array *meta = ddtrace_property_array(&span->property_meta);
     zend_array *metrics = ddtrace_property_array(&span->property_metrics);
 
+    if (get_DD_TRACE_GIT_METADATA_ENABLED()) {
+        ddtrace_inject_git_metadata(&span->property_meta, is_root_span);
+    }
+
```

Comment:
I think we should inject these metadata on span creation. I.e. put it into ddtrace_set_root_span_properties. Children probably should inherit it from the root span (just like service etc. are also inherited) in ddtrace_inherit_span_properties then.

Or when programmatically assigning to root span.